azext_iot/tests/deviceupdate/conftest.py

```diff
@@ -22,8 +22,10 @@
 
 ACCOUNT_RG = settings.env.azext_iot_testrg
 VALID_IDENTITY_MAP = {"system": 1, "user": 1}
-DEFAULT_ADU_RBAC_SLEEP_SEC = 90
-ADU_CLIENT_DTMI = "dtmi:azure:iot:deviceUpdateModel;1"
+DEFAULT_ADU_RBAC_SLEEP_SEC = 120
```

Comment:
Were we seeing RBAC failures with sleep time of 90 seconds? Or was there any other driver for this change?

Reply:
Very little...ADU tests have been pretty stable, but there was a timing issue in this last iteration which motivated the increase. I'll reduce it by half and keep an eye on it.